SU-SWS/stanford_paragraph_types
# Earth 336 callout filmstrip

# READY FOR REVIEW

# Summary
- Changed text fields to text areas.
- Added variants for callout filmstrip

# Needed By (Date)
- 9/7

# Urgency
- Medium

# Steps to Test

1. Checkout this branch and same branch in [stanford_components](https://github.com/SU-SWS/stanford_components/pull/82)
2. clear cache, db update & features import all
3. create complex page with callout filmstrip
4. set the display variant to "above"
5. ensure it looks close and just needs some additional styles

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
